docs/demo/gaiden-css/components/footer/demo.html

```diff
@@ -12,65 +12,79 @@
-      <source srcset="/demo/gaiden-css/images/background-header.jpg" media="(max-width: 767px)">
-      <img src="/demo/gaiden-css/images/background-header-medium.jpg" class="picture__image" alt="Imagem">
-    </picture>
-    <header class="page-banner__content" role="sectionhead">
-      <h1 class="page-title" arial-level="1" role="heading">
-        Encontre os melhores Pedreiros da sua Região
-        <small>Você nos conta o que precisa, nós selecionamos os melhores pedreiros e eles entrarão em contato com você.</small>
-      </h1>
-    </header>
-  </section>
-
-  <footer class="footer footer--sticky">
-    <nav class="footer__menu menu menu--centered">
-      <ul class="container container--middle-items">
-        <li class="col-small-12 col-normal-2"><a class="footer__link" href="#">Quem Somos</a></li>
-        <li class="col-small-12 col-normal-2"><a class="footer__link" href="#">Guia do GetNinjas</a></li>
-        <li class="col-small-12 col-normal-2"><a class="footer__link" href="#">Ajuda</a></li>
-        <li class="col-small-12 col-normal-2"><a class="footer__link" href="#">Termos de Uso</a></li>
-        <li class="col-small-12 col-normal-4"><a href="#" class="hint footer__phone"><img src="/demo/gaiden-css/images/icons/phone.svg" class="hint__icon"> <span class="hint__message">(11) 3791-7315</span></a></li>
-      </ul>
-    </nav>
-
-    <section class="footer__company-info">
-      <div class="container container--middle-items">
-        <div class="col-small-12 col-normal-5 brand footer__brand">
-          <a href="#" class="brand__name">GetNinjas</a>
-        </div>
-        <div class="col-small-12 col-normal-7">
-          <ul class="footer__social-bar">
-            <li><a href="https://www.facebook.com/GetNinjas" class="footer__link"><img src="/demo/gaiden-css/images/icons/facebook.svg"></a></li>
-            <li><a href="https://twitter.com/getninjas" class="footer__link"><img src="/demo/gaiden-css/images/icons/twitter.svg"></a></li>
-            <li><a href="https://plus.google.com/+GetninjasBr" class="footer__link"><img src="/demo/gaiden-css/images/icons/google-plus.svg"></a></li>
-            <li><a href="https://www.linkedin.com/company/getninjas" class="footer__link"><img src="/demo/gaiden-css/images/icons/linkedin.svg"></a></li>
-          </ul>
-        </div>
-      <div class="footer__company-disclaimer" role="presentation">
-        <p>©2011-2017, GetNinjas Serviços de Internet. CNPJ 14.127.813/0001-51</p> - <address >Av. Rebouças, 2472 - Pinheiros São Paulo - SP</address>
+  <footer class="footer">
+    <nav class="footer__wrapper container" role="menubar" aria-label="Menu rodapé">
+      <div class="footer__block col-small-12 col-medium-6 col-normal-6 col-large-4">
+        <span class="footer__title">
+          Serviços
+        </span>
+        <ul class="footer__list footer__list--on-column col-small-6 col-medium-6 col-normal-6  col-large-6" role="menu" aria-label="Menu rodapé - serviços">
```

Comment:
Não precisamos colocar todos os `col-6` já que ele é o mesmo tamanho desde o small.

Reply:
Yes. We need to fix this on our home Sass, on Scorpion project